api/controllers/v1/slack.ts

```diff
@@ -65,8 +80,9 @@ export const DisplayRepoOptions = async (event: APIGatewayEvent): Promise<APIGat
 async function deployRepo(deployable: Array<any>, logger: ILogger, jobRepository: JobRepository, jobQueueUrl) {
   try {
     await jobRepository.insertBulkJobs(deployable, jobQueueUrl);
+    console.error('testing console error logging');
```

Comment:
Should we remove this?

Reply:
Yup, it will be- left it in for Matt to reference regarding his comment on logging !